WooCommerce/WooCommerceScreenshots/Screens/TabNavComponent.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import XCTest
+
+class TabNavComponent: BaseScreen {
+
+    struct ElementStringIDs {
+        static let myStoreTabBarItem = "tab-bar-my-store-item"
+        static let ordersTabBarItem = "tab-bar-orders-item"
+        static let reviewsTabBarItem = "tab-bar-reviews-item"
+        static let settingsButton = "dashboard-settings-button"
```

Comment:
I don't think we need this settingsButton ID string here (not part of this component).

Reply:
Ah, nice catch. Fixed!